src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf

```diff
@@ -2569,7 +2574,7 @@
       </trans-unit>
       <trans-unit id="IDS_FeatureCollectionExpressions">
         <source>collection expressions</source>
-        <target state="translated">výrazy kolekce</target>
+        <target state="needs-review-translation">výrazy kolekce</target>
```

Comment:
><target state="needs-review-translation">výrazy kolekce</target> [](http://example.com/codeflow?start=8&length=64)

This change should be reverted because we are not changing IDS_FeatureCollectionExpressions

Reply:
Applicable to similar changes in other files

Reply:
Done